standalone.md

```diff
@@ -0,0 +1,59 @@
+# Setting up TestGrid
+
+This tutorial goes over setting up your own TestGrid instance in Google Cloud
+Storage.
```

Comment:
```suggestion
This tutorial goes over setting up your own TestGrid instance in [Google Cloud
Storage].
```
`[Google Cloud Storage]: https://cloud.google.com/storage`

might be nice for quick context if anyone hasn't seen the infra